### PR TITLE
give example of Utilities::MPI::Partitioner

### PR DESCRIPTION
**Pull request by davydden**
p.s. output is from a variation of `parallel_partitioner_03.cc`.

### CODE REVIEW

**Comment by davydden** [comment]
screen shot 2018-03-25 at 00 17 34

File: include/deal.II/base/partitioner.h [hunk 1]
```diff
@@ -53,6 +53,14 @@ namespace Utilities
      * information is gathered once when constructing the partitioner, which
      * obviates subsequent global communication steps when exchanging data.
      *
+     * The figure below gives an example of index space $[0,74)$ being split into
+     * four processes.
+     * @image html partitioner.png
+     * Here process 0 will import 5 DoFs from process 1 (first pair of import targets),
+     * which coreponds to the first 3 elements of its import indices. Whereas
+     * proces 2 will import 3 DoFs from process 0, which corresponds to the first
```
Comment (kronbichler) [inline include/deal.II/base/partitioner.h]: process 2

File: include/deal.II/base/partitioner.h [hunk 2]
```diff
@@ -53,6 +53,14 @@ namespace Utilities
      * information is gathered once when constructing the partitioner, which
      * obviates subsequent global communication steps when exchanging data.
      *
+     * The figure below gives an example of index space $[0,74)$ being split into
+     * four processes.
+     * @image html partitioner.png
+     * Here process 0 will import 5 DoFs from process 1 (first pair of import targets),
+     * which coreponds to the first 3 elements of its import indices. Whereas
```
Comment (kronbichler) [inline include/deal.II/base/partitioner.h]: corresponds

**Pull request merged**